pact-lang-api-global.min.js

```diff
@@ -5455,15 +5551,15 @@ const fetchSend = async function(sendCmd, apiHost){
  */
 const fetchLocal = async function(localCmd, apiHost) {
   if (!apiHost)  throw new Error(`Pact.fetch.local(): No apiHost provided`);
-  const {keyPairs, nonce, pactCode, envData, meta, networkId} = localCmd
-  const cmd = prepareExecCmd(keyPairs, nonce, pactCode, envData, meta, networkId);
+  const {keyPairs, nonce, pactCode, envData, meta} = localCmd
```

Comment:
why no networkId?

Reply:
I noticed that /local calls don't need networkId